### PR TITLE
Crate docs for `servers::upd` module

### LINKED ISSUE #271: crate docs: for `servers::udp` mod (UDP tracker)
Parent issue: https://github.com/torrust/torrust-tracker/issues/252

### PR DESCRIPTION
**Pull request by josecelano**
Documentation for the `crate::servers::udp` module. The UDP tracker.

### CODE REVIEW

File: src/servers/udp/mod.rs [hunk 1]
```diff
@@ -1,3 +1,161 @@
...
+//! Refer to the [`bit_torrent`](crate::shared::bit_torrent) module for more
+//! information about the `BitTorrent` protocol.
+//!
+//! Refer to [BEP 15. UDP Tracker Protocol for `BitTorrent`](https://www.bittorrent.org/beps/bep_0015.html)
+//! for more information about the UDP tracker protocol. And to [BEP 41. UDP Tracker Protocol Extensions].
+//!
+//! > **NOTICE**: BEP-41 is not implemented yet.
+//!
+//! > **NOTICE**: we are using the [`aquatic_udp_protocol`](https://crates.io/crates/aquatic_udp_protocol)
+//! crate so requests and responses are handled by it. For the
```
Comment (da2ce7) [inline src/servers/udp/mod.rs]: For the... ?

**Review by da2ce7 — COMMENTED** [review]
Looks Good. :)

**Comment by josecelano** [comment]
Hi @da2ce7 @WarmBeer, this is ready to review. It would be nice if you reviewed the parts you implemented, like:

- `src/servers/udp/connection_cookie.rs`
- `src/servers/apis/server.rs`
- `src/servers/udp/server.rs`
- `src/servers/http/server.rs`

I'm going to continue documenting the `packages` as described [here](https://github.com/torrust/torrust-tracker/issues/252).

**Review by da2ce7 — APPROVED** [review]
Looks Great! Really good work with the UDP Tracker documentation!

**Comment by josecelano** [comment]
ACK 7014a4645d0c71e503e5d789373f6e6bfc04d310

**Pull request merged**